music21/stream/base.py

```diff
@@ -9269,29 +9269,32 @@ def isWellFormedNotation(self):
         >>> s.parts[0].getElementsByClass('Measure').first().isWellFormedNotation()
         True
 
-        >>> s = stream.Score()
+        >>> s2 = stream.Score()
         >>> m = stream.Measure()
-        >>> s.append(m)
-        >>> s.isWellFormedNotation()
+        >>> s2.append(m)
+        >>> s2.isWellFormedNotation()
+        False
+
+        >>> o = stream.Opus([s])
+        >>> o.isWellFormedNotation()
+        True
+        >>> o2 = stream.Opus([s2])
+        >>> o2.isWellFormedNotation()
         False
         '''
+        def allSubstreamsHaveMeasures(testStream):
+            return all(s.hasMeasures() for s in testStream.getElementsByClass('Stream'))
+
         # if a measure, we assume we are well-formed
         if 'Measure' in self.classes:
             return True
         elif 'Part' in self.classes:
             if self.hasMeasures():
                 return True
+        elif 'Opus' in self.classes:
+            return all(allSubstreamsHaveMeasures(s) for s in self.scores)
```

Comment:
need a test on `stream.Opus().isWellFormedNotation()`  should return False (no scores), but given that `all([])` is True, it will return True.